oak_services/proto/http_invocation.proto

```diff
@@ -33,3 +33,9 @@ message HttpInvocation {
 message HttpInvocationSender {
   oak.handle.Sender sender = 1 [(oak.handle.message_type) = ".oak.invocation.HttpInvocation"];
 }
+
+message HttpPipe {
+  HttpInvocation http_invocation = 1;
+  oak.handle.Sender request_sender = 2 [(oak.handle.message_type) = ".oak.encap.HttpRequest"];
+  oak.handle.Receiver response_receiver = 3 [(oak.handle.message_type) = ".oak.encap.HttpResponse"];
```

Comment:
This looks like the "dual invocation" that I mentioned a while ago; it is basically an invocation in which sender and receiver are swapped. I think that should be its own type though, and have a method that creates both ends of the invocation and return them to the caller. Maybe a refactoring for the future anyways (especially as I cannot think of a good name for this object -- maybe `Invocation` vs `InvocationProducer`?)